<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidDeployStepWidget</class>
 <widget class="QWidget" name="AndroidDeployStepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>155</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Qt Deployment</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="ministroOption">
        <property name="toolTip">
         <string>Use the external Ministro application to download and maintain Qt libraries.</string>
        </property>
        <property name="text">
         <string>Use Ministro service to install Qt</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="temporaryQtOption">
        <property name="toolTip">
         <string>Push local Qt libraries to device. You must have Qt libraries compiled for that platform.
The APK will not be usable on any other device.</string>
        </property>
        <property name="text">
         <string>Deploy local Qt libraries to temporary directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="bundleQtOption">
        <property name="toolTip">
         <string>Creates a standalone APK.</string>
        </property>
        <property name="text">
         <string>Bundle Qt libraries in APK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Advanced Actions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="cleanLibsPushButton">
        <property name="text">
         <string>Clean Temporary Libraries Directory on Device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="chooseButton">
        <property name="text">
         <string>Install Ministro from APK</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
